deploy/materialized_view_address.sql

```diff
@@ -0,0 +1,100 @@
+-- Deploy ggircs:materialized_view_address to pg
+-- requires: table_ghgr_import
+
+begin;
+
+create materialized view ggircs_swrs.address as (
+  with x as (
+    select _ghgr_import.xml_file as source_xml,
+           _ghgr_import.id         as ghgr_import_id
+    from ggircs_swrs.ghgr_import as _ghgr_import
+    order by ghgr_import_id asc
+  )
+  select ghgr_import_id, address_details.*
+  from x,
+       xmltable(
+           '//Address[not(ancestor::Stack)]'
+           passing source_xml
+           columns
+                facility_id numeric(1000,0) path './ancestor::Facility/../../ReportDetails/FacilityId[normalize-space(.)]|./ancestor::Contact/ancestor::ReportData/ReportDetails/FacilityId[normalize-space(.)]',
+                organisation_id numeric(1000,0) path './ancestor::Organisation/../../ReportDetails/OrganisationId[normalize-space(.)]|./ancestor::ParentOrganisation/ancestor::ReportData/ReportDetails/OrganisationId[normalize-space(.)]',
```

Comment:
@wenzowski I think facility_id and organisation_id should maybe change to include 'swrs' at the beginning of the name. Isn't this the convention we have been using elsewhere?